libs/chain/src/transaction_serializer.cpp

```diff
@@ -47,17 +47,18 @@ using serializers::MsgPackSerializer;
 using TokenAmount  = Transaction::TokenAmount;
 using ContractMode = Transaction::ContractMode;
 
-const uint8_t MAGIC              = 0xA1;
-const uint8_t VERSION            = 3u;
-const int8_t  UNIT_MEGA          = -2;
-const int8_t  UNIT_KILO          = -1;
-const int8_t  UNIT_DEFAULT       = 0;
-const int8_t  UNIT_MILLI         = 1;
-const int8_t  UNIT_MICRO         = 2;
-const int8_t  UNIT_NANO          = 3;
-const int8_t  CONTRACT_PRESENT   = 1;
-const int8_t  CHAIN_CODE_PRESENT = 2;
-const int8_t  SYNERGETIC_PRESENT = 3;
+constexpr uint8_t MAGIC              = 0xA1;
+constexpr uint8_t VERSION            = 3u;
+constexpr int8_t  UNIT_MEGA          = -2;
+constexpr int8_t  UNIT_KILO          = -1;
+constexpr int8_t  UNIT_DEFAULT       = 0;
+constexpr int8_t  UNIT_MILLI         = 1;
+constexpr int8_t  UNIT_MICRO         = 2;
+constexpr int8_t  UNIT_NANO          = 3;
+constexpr int8_t  CONTRACT_PRESENT   = 1;
+constexpr int8_t  CHAIN_CODE_PRESENT = 2;
+constexpr int8_t  SYNERGETIC_PRESENT = 3;
+constexpr uint8_t IDENTITY_KEY       = 0x4;
```

Comment:
This should actually be called something like: Uncompressed secp256k1 public key or something similar - we can do this in the next PR